class-1

```diff
@@ -0,0 +1,112 @@
+console.log('HI');
+
+
+var a =10;
+console.log(a);
+
+let b =15;
+console.log(b);
+
+const x=5;
+console.log(x);
+
+if(a==10){
```

Comment:
Same here, we can do it as `if (a == 10) {`
It will turn your code looks nice and more readable.